From 21247059a6d5aa78facf6655009af949b1864924 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Thu, 15 Aug 2024 03:05:36 -0500
Subject: [PATCH] fix-cfg_file-setting-in-configure-8.311

---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index e2d35cfc..d052a4a2 100755
--- a/configure
+++ b/configure
@@ -428,7 +428,7 @@ done
     
 # Write the configuation to screen and file, update the pythia8-config script.
 mkdir -p bin;
-sed 's|^CFG_FILE=.*|CFG_FILE='"$PREFIX_SHARE"'/examples/Makefile.inc|;n' \
+sed 's|^CFG_FILE=.*|CFG_FILE='"$PREFIX_SHARE"'/examples/Makefile.inc|' \
     pythia8-config.inc > bin/pythia8-config
 chmod a+x bin/pythia8-config
 echo "---------------------------------------------------------------------"
-- 
2.46.0

