From a54b2043218f20e2ae21186dd890ea2d7de3cd6a Mon Sep 17 00:00:00 2001
From: Christopher Burr <chrisburr73@googlemail.com>
Date: Fri, 5 Jun 2020 12:31:39 +0000
Subject: [PATCH] Fix definition of dlsym in PythiaStdlib.h

---
 include/Pythia8/PythiaStdlib.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/Pythia8/PythiaStdlib.h b/include/Pythia8/PythiaStdlib.h
index b23a38f9..71d49522 100644
--- a/include/Pythia8/PythiaStdlib.h
+++ b/include/Pythia8/PythiaStdlib.h
@@ -33,6 +33,7 @@
 
 // Redefine dlsym to suppress compiler warnings.
 extern "C" void *(*dlsym(void *handle, const char *symbol))();
+extern "C" void *dlsym(void *handle, const char *symbol);
 
 // Stdlib header file for input and output.
 #include <iostream>
-- 
2.26.2

